README.md

```diff
@@ -13,6 +13,7 @@
 ## Steps to run locally
 With Docker
 
+* run this command to create the docker volume - ```docker create volume postgres-data```
```

Comment:
do u need this if you've added it in docker-compose?

Reply:
@hkirat My reasoning is that it's better to create the volume you specified in the compose file to avoid any mount error while running the containers.

Reply:
I commented this as optional command in README.md